epub33/rs/index.html

```diff
@@ -219,6 +219,25 @@ <h4>Relationship to SVG</h4>
 				</section>
 			</section>
 		</section>
+		<section id="sec-rs-conformance">
+			<h3>Reading system conformance</h3>
+
+			<p>Whether a reading system has to support a feature is mentioned at the beginning of its section. To be
+				conformant with this specification, reading systems MUST support all required features as well as all
+				applicable conditionally-required features (e.g., to support image rendering if the reading system has a
```

Comment:
Nit: no hyphen with adverbs (generally -ly), only compound adjectives.
https://www.merriam-webster.com/words-at-play/6-common-hypercorrections-and-how-to-avoid-them (not the first section; ctrl-f for "-ly" and you'll see it, which quotes Chicago, but Chicago requires a subscription so I can't link there directly).

Reply:
Lol, ya, I even called that out in review comments here: https://github.com/w3c/epub-specs/issues/2181#issuecomment-1085187900

I always fall into the trap.